ui/apps/platform/src/test-utils/mocks/@patternfly/react-core.tsx

```diff
@@ -1,6 +1,9 @@
 import * as PFReactCore from '@patternfly/react-core';
 
+// disable because unused debounce might be specified for rest spread idiom.
+/* eslint-disable @typescript-eslint/no-unused-vars */
 const { debounce, ...rest } = jest.requireActual('@patternfly/react-core');
+/* eslint-enable @typescript-eslint/no-unused-vars */
```

Comment:
It has been a while since I looked at this code, but it seems like extracting the `debounce` has no effect here. It is possible this is leftover from a previous code change.

Reply:
Good eyes! I rewrote to simplify spread idiom.